block/manager_test.go

```diff
@@ -94,3 +95,88 @@ func TestIsDAIncluded(t *testing.T) {
 	m.blockCache.setDAIncluded(hash.String())
 	require.True(m.IsDAIncluded(hash))
 }
+
+func TestSubmitBlocks(t *testing.T) {
```

Comment:
```suggestion
func TestSubmitBlocksToDA(t *testing.T) {
```

since we should have a separate test for `SubmitBlocks`